.github/workflows/build-package.yml.template

```diff
@@ -110,15 +110,16 @@ jobs:
         run: |
           (
             cd trigger
-            git tag $CRATE-$VERSION-$TARGET_ARCH
```

Comment:
I think the only thing that's left is to reinstate the old tag format?

Once that's done, push it to the actions branch a couple of times to test it out and then merge to main and keep an eye on it

Reply:
I've reinstalled the old tag format and release format, it should now be backwards compatible.